yaksh/templates/yaksh/grade_user.html

```diff
@@ -87,16 +87,17 @@ <h5 data-toggle="tooltip" title="{{course.name}}">
   <div class="row">
     {% if status == "grade" %}
       {% if users %}
-        <div id="student" class="col-md-3">
-          <ul class="nav nav-pills list-group">
-            {% for user in users %}
-              <li class="nav-item">
-                <a href="{% url 'yaksh:grade_user' quiz_id user.user__id course_id %}" class="list-group-item{% if user.user__id == data.user.id %} active {% endif %}">
-                  {{user.user__first_name}} {{user.user__last_name}}
-                </a>
-              </li>
-            {% endfor %}
-          </ul>
+        <div id="student" class="col-lg-3">
+          <div class="students-list">
+            <select onchange="window.location.href=this.value" class="custom-select">
+              <option value="">Select student</option>
+              {% for user in users %}
+                 <option value="{% url 'yaksh:grade_user' quiz_id user.user__id course_id %}" {% if user.user__id == data.user.id %} selected {% endif %}>
+                   {{user.user__first_name|upper}} {{user.user__last_name|upper}}
```

Comment:
If the name is long, it gets mixed with the table on the right-side. This can be ignored cause the name should be considerably longer to break, which is very rare in a real-life case. Other than that looks good to me